server/pbenchinacan/load_keycloak.sh

```diff
@@ -108,6 +108,7 @@ CLIENT_CONF=$(curl -si -f -X POST "${KEYCLOAK_HOST_PORT}/admin/realms/${REALM}/c
        "directAccessGrantsEnabled": true,
        "serviceAccountsEnabled": true,
        "enabled": true,
+       "attributes": {"post.logout.redirect.uris": "'${KEYCLOAK_REDIRECT_URI}'"},
        "redirectUris": ["'${KEYCLOAK_REDIRECT_URI}'"]}')
```

Comment:
It's interesting that the post-logout redirect is separate from `redirectUris` (I mean, `KEYCLOAK_REDIRECT_URI` is a big wildcard which I would have thought would cover the post-logout redirect...so, apparently, they think this redirect is "special"...  😝).

Reply:
Yes, it is; but if there's any actual documentation of what `redirectUris` really means internally, and whether there's a more specific attribute for *login* redirection, I was unable to locate it anywhere. The documentation is extremely clear that there's a property of the client registry named `redirectUris` which is a list of strings, and that's literally all I could find anywhere. It's really good on "what things are" but lacking even the most rudimentary attempt to describe "what things *mean* or why you'd use them". It's almost maddeningly obtuse.